### ISSUE TITLE
linux-kernel: security updates

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-abbs#660]
```
Until recently, /dev/snd/timer driver was prone to a data race, which
led to uninitialized memory from the kernel heap being copied to the
userspace.
```

Fixed CVE: CVE-2017-1000380.

Patches:

- https://github.com/torvalds/linux/commit/d11662f4f798b50d8c8743f433842c3e40fe3378
- https://github.com/torvalds/linux/commit/ba3021b2c79b2fa9114f92790a99deb27a65b728

@cthbleachbit Please apply these patches before bumping Kernel.

**Comment by cthbleachbit** [comment]
2 fixes are already included in 4.11.5 and 4.9.32, no extra intervention required. Packages updated in `ff65e80` and `ccd7634`

**Comment by MingcongBai** [comment]
Use AOSA-2017-0043.
